server/test/unit/apiTest/authorizeTest.js

```diff
@@ -6,28 +6,16 @@ const sinonChai = require('sinon-chai')
 
 // In-house dependencies
 const { helpers } = require('brave-alert-lib')
-
+const { mockResponse } = require('../../../testingHelpers')
 const api = require('../../../api')
 
 // Configure Chai
 use(sinonChai)
 
 const sandbox = sinon.createSandbox()
 
-function authorize_next() {
-  helpers.log('authorize_next')
-}
-
-function mockRequest(_req) {
-  const req = _req
-  return req
-}
-
-function mockResponse() {
-  const res = {}
-  res.status = sinon.stub().returns(res)
-  res.send = sinon.stub().returns(res)
-  return res
+function authorizeNext() {
+  helpers.log('authorizeNext')
```

Comment:
Good problem-solving to figure out a way to see if the `next` function is called.

A more direct way to do the same thing is to use a Sinon stub. For example:

`function authorizeNext = sandbox.stub()`

and then you'll be able to call methods like `expect(authorizeNext).to.be.called`

Reply:
done!